src/Serilog.Sinks.Console/Sinks/SystemConsole/Output/TextTokenRenderer.cs

```diff
@@ -32,7 +32,7 @@ public TextTokenRenderer(ConsoleTheme theme, string text)
         public override void Render(LogEvent logEvent, TextWriter output)
         {
             var _ = 0;
-            using (_theme.Apply(output, ConsoleThemeStyle.SecondaryText, ref _))
+            using (_theme.Apply(output, ConsoleThemeStyle.TertiaryText, ref _))
```

Comment:
This one is the actual change; the other diff points are caused by the enum item rename.